inscriptions/ISic030011.xml

```diff
@@ -109,9 +113,10 @@
                                 <placeName type="modern">Selinunte</placeName>
                                 <geo>37.5853775, 12.8298666</geo>
                             </origPlace>
-                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" notBefore-custom="-0600" notAfter-custom="-0400" precision="low">6th—5th century BCE</origDate> <!-- check date -->
+                            <origDate datingMethod="#julian" evidence="lettering archaeological-context" cert="low" notBefore-custom="-0475" notAfter-custom="-0450">475—450 BCE</origDate> 
```

Comment:
you can add the information that the dates are from Jordan in plain text: 475—450 BCE (so Jordan on letters)